website/content/specification/syntax/constraints.md

```diff
@@ -14,27 +14,79 @@ TODO: P3: Address issue https://github.com/usnistgov/metaschema/issues/325
+
+Once the *target set* of `<allowed-values>` constraints is validated, the `@allow-other` attribute is used to determine the *expected value set* for a given content value.
+
+The following subsections detail the processing requirements for the `@extension` and `@allow-other` attributes.
+
+### `@extension`
+
+For each `<allowed-values>` constraint, the `@extension` attribute MUST be one of the following values.
+
+- **`none`:** There can be no other matching `<allowed-values>` constraint for the same target value. This is the least permissive option.
+
+- **`model`:** (default) Multiple matching `<allowed-values>` constraints are allowed for the same target value as long as the constraints are defined in the same model. Constraints sourced from outside the model are not allowed.
+
+    All allowed-values constraints declared within a Metaschema model matching the same @target can be combined. If a matching constraint within the model has allow-other="no", then constraints declared externally from the model are not allowed. **This is the implicit default value if no `@extension` is provided.**
+
+- **`external`:** Multiple matching `<allowed-values>` constraints are allowed for the same target value, which can be sourced from within the model or externally through a set of external constraints.
+
+    All allowed-values constraints, declared within the model and externally through an extension, that match the same @target can be combined. This is the most permissive option.
+
+One of the following requirements MUST apply when processing a value's *target set* to validate it.
+
+1. The *target set* MUST contain a single `<allowed-values>` constraint with the `@extension` attribute value `none`.
+1. All `<allowed-values>` constraints in the *target set* MUST have the `@extension` attribute value `model` and originate from a *model* source.
+1. All `<allowed-values>` constraints in the *target set* MUST have the `@extension` attribute value `external` and originate from either a *model* or *extension* source.
+1. An error MUST be raised indicating the *target set* is invalid.
+
+### `@allow-other`
+
+The *expected value set* can be considered *open* or *closed*.
+
+- **open:** In an open set, the actual value can be any value. The *expected value set* provides suggested values.
+- **closed:** In a closed set, the actual value is expected to match a value in the *expected value set*.
+
+For each `<allowed-values>` constraint, the `@allow-other` attribute MUST be one of the following values.
+
+- **`yes`:** Identifies the *expected value set* as *open*, as long as no other `<allowed-values>` constraint in the *target set* has `@allow-other="no"` declared explicitly or implicitly.
+- **`no`:** (default) Identifies the *expected value set* as *closed*. **This is the implicit default value if no `@allow-other` is provided.**
+
+One of the following requirements MUST apply when processing the a value's *target set* to determine the *expected value set*.
```

Comment:
```suggestion
One of the following requirements MUST apply when processing a value's *targeting set* of `<allowed-values>` constraints to determine the *expected value set*.
```

Reply:
I agree with the additions but should we not keep it target set so align with the name of the tag in the syntax itself?